### PR TITLE
Change Travis distribution target from Ubuntu 16.04 to 18.04

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

16.04 is nearly 4 years old now. This PR changes Travis to test a more recent OS. It looks like it may also cut down on the start up time of the tests.

Modifications to the config file are made based on the official Travis update guide. https://docs.travis-ci.com/user/reference/bionic/

#### Link(s) to related issue(s)

* #5253 should be updated with version information following this PR being merged.

### CODE REVIEW

**Comment by johnsaigle** [comment]
Making additional changes based on recommendations from Travis about our config:

Screen Shot 2020-03-02 at 11 25 58

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+os: linux
+dist: bionic
+
+jobs:
   include:
   - php: "7.3"
   - php: "7.4"
 
 services:
+- mysql
```
Comment (kongtiaowang) [inline .travis.yml]: This is not necessary. Docker will create mysql service for testing.

Comment (johnsaigle) [reply .travis.yml]: It has changed for 18.04. It explains in the link

Comment (kongtiaowang) [reply .travis.yml]: "On the Ubuntu 18.04 based environment, to speed up boot time and improve performance we’ve disabled all services by default. Add any services that you want to start to your .travis.yml:  "
Integration testing only needs mysql service from docker system not from travis.

Comment (driusan) [reply .travis.yml]: The link just says they've disabled all services by default (that I see).. I think @kongtiaowang's point is that we don't use the Travis mysql service, it gets launched from docker.

Comment (johnsaigle) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ install:
 - sudo apt-get update
 - sudo apt-get install -y libapparmor1
 - pecl install -f ast-1.0.3
-- sudo apt-get install npm
+- sudo apt-get install nodejs-dev node-gyp libssl1.0-dev npm
```
Comment (driusan) [inline .travis.yml:26]: what are these new dependencies? Are they the same as documented elsewhere in our documentation?

Comment (johnsaigle) [reply .travis.yml:26]: I think this is specific to the way Travis preloads things for Bionic. This was required to get node to stall properly (which in turn installs npm). Without this line, the build will fail

Comment (ridz1208) [reply .travis.yml:26]: @kongtiaowang can you try installing LORIS on a new ubuntu 18.04 machine without these added dependencies and see if it works ?

John is saying these changes are needed for travis but we want to know if they are also needed for regular installs

Comment (driusan) [reply .travis.yml:26]: @kongtiaowang

Comment (ridz1208) [reply .travis.yml:26]: @kongtiaowang please answer comment above

**Comment by cmadjar** [comment]
@kongtiaowang can you check if this is required outside of Travis?

**Comment by kongtiaowang** [comment]
@cmadjar  This is not required outside of Travis. It is only using on Travis. "test-install.loris.ca" is already on ubuntu 18.04, it is not required.

**Pull request merged**